packages/contentstack-audit/src/types/content-types.ts

```diff
@@ -51,6 +51,7 @@ type RefErrorReturnType = {
   uid?: string;
   content_types?: string[];
   title?: string;
+  branches?: string[];
```

Comment:
Can you remove the `branches` key from here? And create a new type and extend `branches` and other keys it there?

Reply:
Removed the branches